data/File.txt

```diff
@@ -0,0 +1,9 @@
+# Guider and FVC image paths
```

Comment:
what is this `File.txt`?  Is this something new?

Reply:
The test was the issue, which I just fixed (in principle) because unlike sdsswork which has a top level mwm folder, ipl-3 and dr19 have a top level spectro folder.

I didn't add the test with `File.txt`, but it isn't real and should be deleted, in my opinion.

Reply:
if someone added `File.txt`, and we don't need, can you delete it?